### PR TITLE
Added `Polkaswap` section

### LINKED ISSUE #110: Polkaswap Category to Match Old Wiki Setup
Hello, 
It has been brought to our attention that segmenting the Polkaswap information could be difficult for new users, within the "Participate in the SORA Economy" section, can we please add a Polkaswap section that includes user guides, from inexperienced to advanced, similar to the structure in the old wiki?
Image attached, 
Thank you. 
IMAGE 2023-07-11 13:53:26

### PR DESCRIPTION
**Pull request by bgodlin**

### CODE REVIEW

**Review by WRRicht3r — APPROVED** [review]

File: src/contribute.md [hunk 1]
```diff
@@ -1 +1 @@
-<!-- @include: ./../CONTRIBUTING.md -->
+<!-- @include: ./explore.md -->
```
Comment (outoftardis) [inline src/contribute.md]: why this change? this is supposed to be the contributing guide for doc repo

Comment (bgodlin) [reply src/contribute.md]: That's a mistake

**Review by outoftardis — APPROVED** [review]
